dot/peerset/peerset.go

```diff
@@ -666,15 +666,14 @@ func (ps *PeerSet) disconnect(setIdx int, reason DropReason, peers ...peer.ID) e
 }
 
 // start handles all the action for the peerSet.
-func (ps *PeerSet) start(aq chan action, stopCh chan struct{}) {
+func (ps *PeerSet) start(ctx context.Context, aq chan action) {
```

Comment:
Rename `aq` to `actionQueue` please; also maybe restrict to a read only / write only channel if you can

Reply:
the `PeerSet.actionQueue` already is a `<-chan action` (read only channel)

Reply:
So you could restrict the argument to be `actionQueue chan<- action` too :wink: